.github/workflows/buildkit-daily.yml

```diff
@@ -84,6 +84,11 @@ jobs:
           tags: |
             type=raw,latest
             type=raw,${{steps.get-tarball-date.outputs.result}}
+          labels: |
+            org.opencontainers.image.title=AOSC OS BuildKit
+            org.opencontainers.image.description=BuildKit is essentially a non-bootable AOSC OS Base with full development support.
```

Comment:
Drop the preceding notes.

"AOSC OS base system with full development support."

Reply:
```suggestion
            org.opencontainers.image.description=AOSC OS Base with full development support.
```